jinahub/indexers/searcher/AnnoySearcher/__init__.py

```diff
@@ -84,7 +87,10 @@ def search(self, docs: DocumentArray, parameters: Dict, **kwargs):
             )
             for idx, dist in zip(indices, dists):
                 match = Document(id=self._ids[idx], embedding=self._vecs[idx])
-                match.scores['distance'] = 1 / (1 + dist)
+                if self.reverse_score:
+                    match.scores['similarity'] = 1 / (1 + dist)
+                else:
+                    match.scores['distance'] = dist
```

Comment:
the name of the key should relate to the metric

Reply:
In ranker when we want to extract the scores values, should I specify the keys like 'distance' 'similarity'? Now in customized ranker I have something like this, if I have different names, How can I get the values?

`chunk_match_list.sort(key=lambda m: -m.scores['distance'].value)`

Reply:
done